scripts/utils.js

```diff
@@ -165,7 +169,8 @@ const updateDenoImportMap = async (importMapPath, nextVersion) => {
       let [packageName, importPath] =
         getPackageNameFromImportSpecifier(importName);
 
-      return remixPackagesFull.includes(packageName)
+      return remixPackagesFull.includes(packageName) &&
+        importName !== "@remix-run/deno"
```

Comment:
`@remix-run/deno` code is already a Deno module, so just get types for it directly from `node_modules/`

https://github.com/remix-run/remix/blob/d5ca496c47ff84c02509b0957e54869de014521e/templates/deno/.vscode/resolve_npm_imports.json#L10-L11